pkg/api/types.go

```diff
@@ -1205,6 +1221,13 @@ type KeyToPath struct {
 	Mode *int32
 }
 
+// Local represents a directory with specific topology contraints
+type LocalVolumeSource struct {
+	// The full path to the volume on the node
+	// For alpha, this path must be a file-based volume
+	Path string
```

Comment:
3) looks quite ugly, so 2 looks the best from the list. Can you add a comment that `Path` is going to be reused both for a dir and device so the one who's going to approve the API changes (bgrant?) is aware of it?